input_dictionary.json

```diff
@@ -750,7 +803,130 @@
\n\nMore details on the theoretical foundations of the estimator can be found in the corresponding papers:\n\n[[FLP08]](https://link.springer.com/chapter/10.1007/978-3-540-85174-5_16) Luk Bettale, Jean-Charles Faugère, and Ludovic Perret. Cryptanalysis of MinRank. [[preprint]](https://www-polsys.lip6.fr/~jcf/Papers/crypto08.pdf)\n\n\n[[BBC+20]](https://link.springer.com/chapter/10.1007/978-3-030-64837-4_17) Magali Bardet, Maxime Bros, Daniel Cabarcas, Philippe Gaborit, Ray Perlner, Daniel Smith-Tone, Jean-Pierre Tillich, and Javier Verbel . Improvements of Algebraic Attacks for Solving the Rank Decoding and MinRank Problems. [[eprint]](https://arxiv.org/pdf/2002.08322.pdf)\n\n[[VBC+19]](https://link.springer.com/chapter/10.1007/978-3-030-25510-7_10) Javier Verbel, John Baena, Daniel Cabarcas, Ray Perlner and Daniel Smith-Tone. On the Complexity of “Superdetermined” Minrank Instances. [[eprint]](https://eprint.iacr.org/2019/731.pdf)\n\n[[Cou01]](https://link.springer.com/chapter/10.1007/3-540-45682-1_24) Nicolas Courtois . Efficient Zero-Knowledge Authentication Based on a Linear Algebra Problem MinRank. [[eprint]](https://www.iacr.org/archive/asiacrypt2001/22480404.pdf)\n\n[[GC00]](https://link.springer.com/content/pdf/10.1007/3-540-44448-3_4.pdf) Louis Goubin and Nicolas Courtois. Cryptanalysis of the TTM cryptosystem. \n\n[[BBB+23]](https://link.springer.com/article/10.1007/s10623-023-01265-x) Magali Bardet, Pierre Briaud, Maxime Bros, Philippe Gaborit, and Jean-Pierre Tillich. Revisiting algebraic attacks on MinRank and on the rank decoding problem. [[eprint]](https://eprint.iacr.org/2022/1031.pdf)\n\n[[GD22]](https://link.springer.com/chapter/10.1007/978-3-031-22944-2_15) Hao Guo and Jintai Ding. Algebraic Relation of Three MinRank Algebraic Modelings. [[eprint]](https://eprint.iacr.org/2022/1226.pdf)\n\n[[KS99]](https://link.springer.com/chapter/10.1007/3-540-48405-1_2) Aviad Kipnis and Adi Shamir. Cryptanalysis of the HFE Public Key Cryptosystem by Relinearization.\n\n[[BB22]](https://link.springer.com/chapter/10.1007/978-3-031-17234-2_6) Author name. Improvement of Algebraic Attacks for Solving Superdetermined MinRank Instances. [[eprint]](https://arxiv.org/abs/2208.01442)\n\n[[FSS10]](https://members.loria.fr/PJSpaenlehauer/data/papers/FauSafSpa10a.pdf) Jean-Charles Faugère , Mohab Safey El Din, and Pierre-Jean Spaenlehauer. Computing Loci of Rank Defects of Linear Matrices using Gröbner Bases and Applications to Cryptology.\n\n[[FSS13]](https://www.sciencedirect.com/science/article/pii/S0747717113000485) Author name. On the Complexity of the Generalized MinRank Problem. [[eprint]](https://arxiv.org/abs/1112.4411)\n\n[[GNS23]](https://dl.acm.org/doi/fullHtml/10.1145/3597066.3597077) Sriram Gopalakrishnan, Vincent Neiger, and Mohab Safey El Din. Refined F5 Algorithms for Ideals of Minors of Square Matrices. [[eprint]](https://arxiv.org/abs/2302.05375#:~:text=We%20consider%20the%20problem%20of,characteristic%20zero%20or%20large%20enough.)",
```

Comment:
For the following paper are no authors listed:
- On the Complexity of the Generalized MinRank Problem: Jean-Charles Faugère, Mohab Safey El Din, Pierre-Jean Spaenlehauer
- Improvement of algebraic attacks for solving superdetermined MinRank instances: Magali Bardet, Manon Bertin

Reply:
There is only a template: "Authors name."

Reply:
Done